llvm-external-projects/iree-dialects/BUILD.bazel

```diff
@@ -666,7 +667,7 @@ cc_library(
 # Tools
 ################################################################################
 
-cc_binary(
+iree_compiler_cc_binary(
```

Comment:
Quoting the README at https://github.com/openxla/iree/tree/main/llvm-external-projects:

> Projects in this tree are targeted for interop with upstream LLVM/MLIR and related projects. They follow LLVM standards, build system, API design and packaging conventions.

This directory shouldn't take a dependency on other IREE code.

Reply:
Answers my question. Thanks!